AEP/aep-00001.md

```diff
@@ -115,11 +116,17 @@ In a way this section can be thought of as a breakout section of the Rationale s
 
 13. **Copyright/license** – Each new AEP must be placed under a dual license of public domain and CC0-1.0-Universal (see this AEP for an example).
 
-# AEP Formats and Templates
+## AEP Formats and Templates
 
-AEPs are UTF-8 encoded text files using the [GitHub-flavored markdown](https://github.github.com/gfm/) format. Using undocumented Markdown syntax which "works" at one point with GitHub is discouraged, as it could break later without notice. Further specifications can be found in the [GitHub documentation](https://docs.github.com/en/get-started/writing-on-github).
+AEPs are UTF-8 encoded, english text files using the [GitHub-flavored markdown](https://github.github.com/gfm/) format. Using undocumented Markdown syntax which "works" at one point with GitHub is discouraged, as it could break later without notice. Further specifications can be found in the [GitHub documentation](https://docs.github.com/en/get-started/writing-on-github). The AEP must pass the linting executed via the following command at the root of the repository:
```

Comment:
```suggestion
AEPs are UTF-8 encoded, english text files using the [GitHub-flavored markdown](https://github.github.com/gfm/) format. Using undocumented Markdown syntax which "works" at one point with GitHub is discouraged, as it could break later without notice. Further specifications can be found in the [GitHub documentation](https://docs.github.com/en/get-started/writing-on-github). The AEP must pass the linter executed via the following command at the root of the repository:
```